OSIsoft_Demo/src/JSONArray.java

```diff
@@ -0,0 +1,891 @@
+
```

Comment:
Should we put this stuff in a subdir to separate it out from the main code?

Reply:
added a commit to the end to put these files in the JSON folder